api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -72,6 +72,27 @@ message HTTPFaultConfig {
   }
 }
 
+// The configuration of a Redis fault.
+message RedisFaultConfig {
+  // The targeting of the fault describing what redis requests are being considered for faults.
+  RedisFaultTargeting fault_targeting = 1 [ (validate.rules).message.required = true ];
+
+  oneof fault {
+    option (validate.required) = true;
+
+    // The error fault responds to redis commands with an error.
+    RedisErrorFault error_fault = 2;
+    // The latency fault delays redis commands.
+    LatencyFault latency_fault = 3;
+  }
+}
+
+// The definition of a redis error fault.
+message RedisErrorFault {
+  // The percentage of requests the fault should be applied to.
+  FaultPercentage percentage = 1 [ (validate.rules).message.required = true ];
+}
+
 // The definition of an abort fault.
```

Comment:
Can skip in this PR but this should probably now speak this to this being specifically about HTTP aborts